bims/templates/main_base.html

```diff
@@ -65,7 +65,7 @@
                         <div class="bims-title-acc col-lg-6">BIODIVERSITY <br/>INFORMATION MANAGEMENT SYSTEM</div>
                     </div>
                      {% if user.is_authenticated %}
-                        <li><a class="nav-link js-scroll-trigger" id="navbar-profile" href="/profile/">PROFILE</a></li>
+                        <li><a class="nav-link js-scroll-trigger" id="navbar-profile" href="{% url 'profile_detail' user.username %}">PROFILE</a></li>
```

Comment:
this is not a good fix, the old profile url still causing error page. maybe just redirect from the backend instead.